code/modules/surgery/organs/parasites.dm

```diff
@@ -0,0 +1,35 @@
+/obj/item/organ/internal/body_egg/spider_eggs
+	name = "spider eggs"
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "eggs"
+	var/stage = 1
+
+/obj/item/organ/internal/body_egg/spider_eggs/on_life()
+	if(stage < 5 && prob(3))
```

Comment:
Yeah, but there's a reason for that; so the stage goes up when the person is dead, and then it gibs out.

With aliens, it's largely to prevent suicide meta and with changeling body eggs it's because it only works on corpse.

For these though? I intentionally wanted it to stop advancing if the person died.